### ISSUE TITLE
va value not appropriate in map_block

### ISSUE THREAD

**Issue by CookieLau** [Michael-Steven/BUAA_OS_2019#1]
https://github.com/Michael-Steven/BUAA_OS_2019/blob/60a0e523f8ec295f5176fddeddba1cb97021eb18/fs/fs.c#L77

Maybe va=0 while executing this line, maybe change to diskaddr(blockno) is better

**Comment by Michael-Steven** [comment]
Yes, it is a bug here. And it has been fixed. Thanks for telling me of it!
